docs/api-docs/in-depth-guides/adding-customer.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Adding Customer to a Tracking Request
+---
+
+## Why you would want to add a party to a tracking request?
+
+Adding a party to a tracking request allows you to associate customer information with the tracking request. This can help in organizing and managing your shipments more effectively.
```

Comment:
I think we should make it clearer that the customer added to the tracking request will be assigned to the shipment when it is created - just like ref numbers and tags.

Also maybe introduce the 'party' concept. Something like - "The party represents different parties of the shipment. Currently you can add your customer (typically the BCO) as a party to any shipment"

Reply:
I think it's too early for `party` concept. There is only party for customers now.